### DISCUSSION TITLE
Does cocotb have an active support community?

### DISCUSSION THREAD

**Discussion by ascesiumastro** [cocotb/cocotb#3512] [Q&A] [answered]
I pre-emptively apologize for the silliness of the question.

We are currently looking into integrating cocotb within one of our main projects verification flow, and we just wanted to know whether cocotb has an active support community at this point in time. My understanding by looking at the forum is that the answer would be yes, but I wanted to get your input irrespectively.

**Comment by ktbarrett** [comment]
The people here will try to help, but we are all volunteers, so there is no guarantee we will help or how swiftly we can help.

**Reply by imphil** [reply, marked as answer]
In addition, there are companies you can pay to get support if you'd like to get some form of SLA in place, just reach out and we can connect you.

**Reply by ascesiumastro** [reply]
Many thanks everyone. I feel my question has been answered!
